packages/fxa-settings/src/pages/Signup/container.tsx

```diff
@@ -82,6 +82,7 @@ const SignupContainer = ({
   const beginSignupHandler: BeginSignupHandler = useCallback(
     async (email, password, options) => {
       options.verificationMethod = 'email-otp';
+      options.keys = isOAuthIntegration(integration);
       try {
         const { authPW, unwrapBKey } = await getCredentials(email, password);
         const { data } = await beginSignup({
```

Comment:
It doesn't seem like email is actually an input on this mutation?

Reply:
Email is an expected input in the `SignUpInput` type